tests/test_cliCmdConfigValidate.py

```diff
@@ -25,16 +25,25 @@
 
 import click
 import click.testing
-import os
 import unittest
 
 from lsst.daf.butler.cli import butler
+from lsst.daf.butler.cli.cmd import config_validate
+from lsst.daf.butler.tests import CliCmdTestBase
 
 
-TESTDIR = os.path.abspath(os.path.dirname(__file__))
+class ValidateTest(CliCmdTestBase):
 
+    defaultExpected = dict()
 
-class Suite(unittest.TestCase):
+    command = config_validate
+
+    def test_help(self):
```

Comment:
This definitely looks like it can go in the base class. It's identical everywhere.

Reply:
I was having trouble with putting test_ functions in the base class, which is abstract. But making the most-derived class derive from the command test base and from TestCase works well.